### PR TITLE
Redesign (landing v2)

### PR DESCRIPTION
**Pull request by deansheather**
Tasks:
- [x] Responsivify
- [x] Fix underlines under FAQ titles when not hovering (I didn't notice those when I pushed)
- [x] Add donation links (PGP signed cleartext file on the CDN)
- [x] Update README.md
- [ ] Merge

### CODE REVIEW

**Comment by TriggerRimfire** [comment]
checking it, please give me a few hours

File: src/scss/partials/_content-box.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
+.content-box {
+  height: 100%;
+  padding: 200px 100px 0 100px;
```
Comment (TriggerRimfire) [inline src/scss/partials/_content-box.scss]: Please don't set paddings this way. Use scalable units.

File: src/scss/partials/_faq.scss [hunk 2]
```diff
@@ -0,0 +1,37 @@
+.faq-box {
+  width: 50%;
+
+  > h3 {
+    margin: 30px 0 0 0;
```
Comment (TriggerRimfire) [inline src/scss/partials/_faq.scss]: Margins can't have static units either, please use scalable units.

File: src/scss/partials/_faq.scss [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      display: inline-block;
+      position: relative;
+
+      &:hover:before {
+        display: block;
+      }
+
+      &:before {
+        padding-right: 13px;
+        position: absolute;
```
Comment (TriggerRimfire) [inline src/scss/partials/_faq.scss]: Please avoid using absolute/relative positioning where flexbox is preferred.

File: src/scss/partials/_header.scss [hunk 4]
```diff
@@ -0,0 +1,19 @@
+header {
+  max-width: 660px;
+  margin-bottom: 50px;
+  line-height: 1;
```
Comment (TriggerRimfire) [inline src/scss/partials/_header.scss]: Please don't use line-height where other methods can be used.

**Comment by TriggerRimfire** [comment]
Also take note that you're repeating yourself several times. Set base styles first, make specific changes by ID later.

File: src/scss/partials/_header.scss (hunk 4 above)
Comment (deansheather) [inline src/scss/partials/_header.scss]: Using line-height to reduce the height of the line, not to vertical align.

File: src/scss/partials/_faq.scss (hunk 3 above)
Comment (deansheather) [inline src/scss/partials/_faq.scss]: Absolute positioning is used here to create an "anchor" icon `:before` FAQ title links.

**Comment by deansheather** [comment]
[Redo owo's house style & web design](https://trello.com/c/zV4NbpLM/4-redo-owo-s-house-style-web-design)

**Comment by deansheather** [comment]
Ready to merge.

File: src/scss/partials/_content-box.scss (hunk 1 above)
Comment (deansheather) [inline src/scss/partials/_content-box.scss]: Added breakpoints that use `vw`s for horizontal padding. Top padding is static (per breakpoint), though, as it's a part of the design.

**Comment by jakeoid** [comment]
Why is there Javascript sitting outside of a document?

https://this.is-a.dog/1febc4.png

**Comment by deansheather** [comment]
What do you mean @jakeoid? That `<script>` tag which I assume you're talking about is clearly inside of the document. It's just not compressed.

**Pull request merged**